Android JCenter - build failure

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':react-native-crash-tester:verifyReleaseResources'.
> Could not resolve all task dependencies for configuration ':react-native-crash-tester:releaseRuntimeClasspath'.
   > Could not resolve com.facebook.react:react-native:+.
     Required by:
         project :react-native-crash-tester
      > Failed to list versions for com.facebook.react:react-native.
         > Unable to load Maven meta-data from https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml.
            > Could not HEAD 'https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml'. Received status code 502 from server: Bad Gateway
```

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -2,7 +2,7 @@ buildscript {
     if (project == rootProject) {
         repositories {
             google()
-            jcenter()
+            mavenCentral()
         }
 
         dependencies {
@@ -48,7 +48,7 @@ repositories {
         url("$rootDir/../node_modules/react-native/android")
     }
     google()
-    jcenter()
+    mavenCentral()
 }
 
 dependencies {

--- a/example/android/build.gradle
+++ b/example/android/build.gradle
@@ -38,7 +38,6 @@ allprojects {
             }
         }
         google()
-        jcenter()
         maven { url 'https://www.jitpack.io' }
     }
 }

